Libraries/Components/Touchable/Touchable.js

```diff
@@ -312,6 +314,15 @@ var TouchableMixin = {
     this.pressOutDelayTimeout && clearTimeout(this.pressOutDelayTimeout);
   },
 
+  propTypes: {
```

Comment:
And if you stick with the duplication, might as well nuke the propTypes here in the mixin since it's not complete and could be confusing for people.